mm/2s2h/Rando/Logic/Regions/West.cpp

```diff
@@ -122,26 +122,9 @@ static RegisterShipInitFunc initFunc([]() {
     Regions[RR_GREAT_BAY_COAST] = RandoRegion{ .sceneId = SCENE_30GYOSON,
         .checks = {
             CHECK(RC_GREAT_BAY_COAST_FISHERMAN_MINIGAME, RANDO_EVENTS[RE_CLEARED_GREAT_BAY_TEMPLE] && (HAS_ITEM(ITEM_HOOKSHOT) || CAN_USE_MAGIC_ARROW(ICE))),
-            CHECK(RC_GREAT_BAY_COAST_OWL_STATUE, CAN_USE_SWORD),
             CHECK(RC_GREAT_BAY_COAST_MIKAU, CAN_PLAY_SONG(HEALING)),
```

Comment:
I forgot about this one because the cutscene skip gets around it. Can you make this change, and with it, modify `SkipPushingMikau` to not run if `IS_RANDO && !CAN_USE_ABILITY(SWIM)`?
```suggestion
            CHECK(RC_GREAT_BAY_COAST_MIKAU, CAN_USE_ABILITY(SWIM) && CAN_PLAY_SONG(HEALING)),
```